crates/interpreter/src/instructions/macros.rs

```diff
@@ -178,59 +174,57 @@ macro_rules! pop_top {
 }
 
 macro_rules! push_b256 {
-	( $interp:expr, $( $x:expr ),* ) => (
-		$(
-			match $interp.stack.push_b256($x) {
-				Ok(()) => (),
-				Err(e) => {
-                    $interp.instruction_result = e;
-                    return
-                },
-			}
-		)*
-	)
+	($interp:expr, $($x:expr),* $(,)?) => ($(
```

Comment:
this now lets you push multiple values? or what is the new `$(,)?` for?

Reply:
Just allows trailing comma.